#!/usr/bin/env python
PACKAGE = "navigation_collision_checker"

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

gen.add("elev_diff_threshold", double_t, 0, "Obstacle diff threshold [m]", 0.15, 0.0, 10.0)
gen.add("pose_height_offset", double_t, 0, "Pose height offset [m]", -0.02, -10.0, 10.0)
gen.add("footprint_x", double_t, 0, "Footprint x offset [m]", 0.2, 0.0, 10.0)
gen.add("footprint_y", double_t, 0, "Footprint y offset [m]", 0.2, 0.0, 10.0)

exit(gen.generate(PACKAGE, "navigation_collision_checker", "PathCollisionChecker"))
